### PR TITLE
Task three

### PR DESCRIPTION
**Pull request by Dylan-Petok**

### CODE REVIEW

File: templates/search_movies.html [hunk 1]
```diff
@@ -3,7 +3,22 @@
...
+{% else %}
 <h1 class="mb-5">Search Movie Ratings</h1>
 <p class="mb-3">Search for a movie rating below</p>
+<form action="{{ url_for('search_movies') }}" method="get">
+    <input type="text" name="movie_title" placeholder="Enter movie title" required>
+    <input type="submit" value="Search">
+</form>
+{% endif %}
+
 <!-- TODO: Feature 3 -->
```
Comment (Cj-6) [inline templates/search_movies.html:23]: remove this

File: src/repositories/movie_repository.py [hunk 2]
```diff
@@ -13,6 +13,7 @@ class MovieRepository:
 
         def __init__(self) -> None:
             self._db: dict[int, Movie] = {}
+            self._db[1] = Movie(1, "Test Movie", "Test Director", 5)
```
Comment (Cj-6) [inline src/repositories/movie_repository.py]: Make sure to remove this per the instructions

File: templates/search_movies.html [hunk 3]
```diff
@@ -3,7 +3,22 @@
...
+        <input type="submit" value="Go back to search">
+    </form>
+{% else %}
 <h1 class="mb-5">Search Movie Ratings</h1>
 <p class="mb-3">Search for a movie rating below</p>
+<form action="{{ url_for('search_movies') }}" method="get">
+    <input type="text" name="movie_title" placeholder="Enter movie title" required>
+    <input type="submit" value="Search">
+</form>
+{% endif %}
```
Comment (Cj-6) [inline templates/search_movies.html:21]: Remove this

**Review by Cj-6 — COMMENTED** [review]
Everything looks great, Good job!

File: templates/search_movies.html (hunk 3 above)
Comment (Cj-6) [reply templates/search_movies.html:21]: just line 21 sorry

Comment (Cj-6) [reply templates/search_movies.html:21]: wait nevermind

**Review by bmulli12 — APPROVED** [review]
Looks good

**Pull request merged**